gradle.properties

```diff
@@ -9,8 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1024m
-
+#org.gradle.jvmargs=-Xmx1024m
+org.gradle.jvmargs=-Xmx4096m -XX:MaxPermSize=4096m -XX:+HeapDumpOnOutOfMemoryError
```

Comment:
Why enable HeapDumpOnOutOfMemoryError?

Also, do we need both MaxPermSize & Xmx?

Reply:
I think we should use it as if heap runs out of memory it gives us the error and also dumps the memory out

Reply:
What would we use it for? It's probably a really expensive operation, and the memory dump presumably uses a large amount of disk space. We probably shouldn't enable it if it isn' tneeded.

I'm actually wondering if all we need here is the 'Xmx' setting.

Reply:
ok, I'm using then only -Xmx